constants/index.js

```diff
@@ -0,0 +1,2 @@
+export const regularRole = 2;
```

Comment:
It's a good practice to use capital letters for constants so you can differentiate them better